src/components/combobox/combobox.e2e.ts

```diff
@@ -213,33 +198,29 @@ describe("calcite-combobox", () => {
 
   it("should show correct max items after selection", async () => {
     const page = await newE2EPage();
-
     const maxItems = 6;
-
-    await page.setContent(`
-    <calcite-combobox label="custom values" allow-custom-values placeholder="placeholder" max-items="6">
-      <calcite-combobox-item value="Sequoia" disabled text-label="Sequoia"></calcite-combobox-item>
-      <calcite-combobox-item value="Douglas Fir" text-label="Douglas Fir"></calcite-combobox-item>
-      <calcite-combobox-item value="Daffodil" text-label="Daffodil"></calcite-combobox-item>
-      <calcite-combobox-item value="Black Eyed Susan" text-label="Black Eyed Susan"></calcite-combobox-item>
-      <calcite-combobox-item value="Nasturtium" text-label="Nasturtium"></calcite-combobox-item>
-      <calcite-combobox-item value="Birds" text-label="Birds"></calcite-combobox-item>
-      <calcite-combobox-item value="Reptiles" text-label="Reptiles"></calcite-combobox-item>
-      <calcite-combobox-item value="Amphibians" text-label="Amphibians"></calcite-combobox-item>
-      <calcite-combobox-item value="Rocks" text-label="Rocks"></calcite-combobox-item>
-      <calcite-combobox-item value="Insects" text-label="Insects"></calcite-combobox-item>
-      <calcite-combobox-item value="Rivers" text-label="Rivers"></calcite-combobox-item>
-    </calcite-combobox>
-    `);
-    await page.waitForChanges();
-
-    const element = await page.find("calcite-combobox");
+    await page.setContent(
+      html`
+        <calcite-combobox label="custom values" allow-custom-values placeholder="placeholder" max-items="6">
+          <calcite-combobox-item value="Sequoia" disabled text-label="Sequoia"></calcite-combobox-item>
+          <calcite-combobox-item value="Douglas Fir" text-label="Douglas Fir"></calcite-combobox-item>
+          <calcite-combobox-item value="Daffodil" text-label="Daffodil"></calcite-combobox-item>
+          <calcite-combobox-item value="Black Eyed Susan" text-label="Black Eyed Susan"></calcite-combobox-item>
+          <calcite-combobox-item value="Nasturtium" text-label="Nasturtium"></calcite-combobox-item>
+          <calcite-combobox-item value="Birds" text-label="Birds"></calcite-combobox-item>
+          <calcite-combobox-item value="Reptiles" text-label="Reptiles"></calcite-combobox-item>
+          <calcite-combobox-item value="Amphibians" text-label="Amphibians"></calcite-combobox-item>
+          <calcite-combobox-item value="Rocks" text-label="Rocks"></calcite-combobox-item>
+          <calcite-combobox-item value="Insects" text-label="Insects"></calcite-combobox-item>
+          <calcite-combobox-item value="Rivers" text-label="Rivers"></calcite-combobox-item>
+        </calcite-combobox>
+      `
+    );
+    const input = await page.find("calcite-combobox >>> input");
     const openEvent = page.waitForEvent("calciteComboboxOpen");
-    await element.click();
+    await input.click();
```

Comment:
Does this need to target the `input`? Clicking on the combobox should work too, no? 

This also applies to any other _new/modified_ tests with a similar setup.